build.gradle.kts

```diff
@@ -415,9 +414,8 @@ val updateThirdPartyLicenses by tasks.registering {
     dependsOn(tasks.downloadLicenses)
     doLast {
         javaexec {
-            main = "-jar"
+            classpath(projectDir.resolve("gradle/tools/license-third-party-tool-2.0.jar"))
```

Comment:
Just noticing that some other projects use the version 3.0. Maybe you can correct them later.